src/styles/utils/_variables.scss

```
@@ -0,0 +1,5 @@
+$day-size: 100px;
+$day-color: #eee;
+$day-gap: 1px;
+$calendar-padding: 10px;
+$days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
```

pure editorial, yet it's more readable to make a columnar:
```suggestion
$days:
  "mon" 0,
  "tue" 1,
  "wed" 2,
  "thu" 3,
  "fri" 4,
  "sat" 5,
  "sun" 6;
```